src/main/network/router/mod.rs

```diff
@@ -23,10 +28,17 @@ impl Router {
     }
 
     // Return true if the router changed from empty to non-empty.
+    // TODO: This will eventually not return anything once we have
+    // PacketDevice signaling implemented in rust.
+    // Rob: Coming in future PR.
     fn push(&mut self, packet: Packet) -> bool {
+        self.push_inner(packet, Worker::current_time().unwrap())
+    }
+
+    fn push_inner(&mut self, packet: Packet, now: EmulatedTime) -> bool {
```

Comment:
I think some of these functions in `Router` should be `pub` so they can be accessed outside of the router module?